minifi/minifi-docker/dockerhub/Dockerfile

```diff
@@ -16,13 +16,13 @@
 # under the License.
 #
 
-FROM openjdk:8-jre-alpine
+FROM eclipse-temurin:8-jre
```

Comment:
I think it is good this way just wanted to double check whether it would make sense to pin the exact version as this way we get the latest and we can get "silent" java updates which could cause problems (at least I faced an issue on a previous project because of this). Again I think we are good and this is how it was earlier just wanted to double check.

Reply:
Yes I think it's beneficial to have the latest version for every build even though as you said there is a risk that a new Java version will cause issues. Would keep it as it is if no objections.

Reply:
I think I'm ok with this given it should not be being built frequently. I noticed the same about the `apt-get update && ...` command, but on the whole, I think the benefits are worth it. We can revisit if it becomes a problem.